### DISCUSSION TITLE
Running build tasks on a PR "on-demand"

### DISCUSSION THREAD

**Discussion by fernap3** [community/community#25854] [Actions]
<p>I am setting up an environment where feature branches must pass the status check suite (using branch protection) before being committed to master.  However I would prefer not to run the whole suite on every commit to the feature branch.  Is there a suggested method to allow branch owners to manually trigger the status check suite?  I was thinking I’d be able to do it by triggering a workflow when a user comments on a PR Conversation with a specific message format, but it doesn’t appear there is a hook for PR Conversation comments.</p>
<p>Thanks!</p>
<p>Peter</p>

**Comment by oldskool** [comment, marked as answer]
<p>The <a href="https://help.github.com/en/articles/events-that-trigger-workflows#issue-comment-event-issue_comment" rel="nofollow noopener">issue_comment</a> event type seems to be what you’re looking for.</p>
<pre><code>on:
  issue_comment:
    types: [created]

jobs:
foo:
runs-on: ubuntu-latest

steps:
      - name: run-something
        if: github.event.issue.comment.body == "Github, OK to test"
        run: &lt;do your stuff here&gt;
</code></pre>
<p>Also see my answer in <a href="https://github.community/t/running-github-actions-on-prs-after-explicit-approval/16329/2">this thread</a>. You seem to be looking for the same thing.</p>

**Comment by fernap3** [comment]
<p>Thanks!  I also didn’t realize PR comments are “issue comments” :slight_smile:</p>
<p>Ultimately it would be great to see Actions support a UI for triggering parameterized builds on-demand.</p>
